lldb/test/API/lang/swift/anonymous-clang-types/TestSwiftAnonymousClangTypes.py

```diff
@@ -0,0 +1,39 @@
+import lldb
+from lldbsuite.test.decorators import *
+import lldbsuite.test.lldbtest as lldbtest
+import lldbsuite.test.lldbutil as lldbutil
+import os
+import unittest2
+
+
+class TestSwiftAnonymousClangTypes(lldbtest.TestBase):
+    @skipUnlessDarwin
```

Comment:
This test does not look darwin-specific at all? Plain C types with anonymous members should also work in Linux, shouldn't they?

Reply:
Oops. This was copied over.